packaging/installer/kickstart.sh

```diff
@@ -89,6 +89,21 @@ main() {
   progress "Using ${tmpdir} as a temporary directory."
   cd "${tmpdir}" || fatal "Failed to change current working directory to ${tmpdir}." F000A
 
+  if [ -n "${INSTALL_VERSION}" ]; then
+    if echo "${INSTALL_VERSION}" | grep -E -o "^[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+$" > /dev/null 2>&1; then
+      NEW_SELECTED_RELEASE_CHANNEL="stable"
+    else
+      NEW_SELECTED_RELEASE_CHANNEL="nightly"
+    fi
+
+    if [ "${NETDATA_ONLY_STATIC}" = 1 ] || [ "${NETDATA_ONLY_BUILD}" = 1 ]; then
```

Comment:
Do we only support manually specifying a version if we have also been asked to use a specific build type?

If not, then this check probably needs to be significantly more complicated, and/or be done in the specific code paths for each install type.

Reply:
I am not sure I understand your question. 
Basically, the build type is redundant, if we specify the version. If it is set, we throw the warning and ignore it, if it's not matching the version the user is trying to install.

Reply:
The way this check is written, it will only be evaluated if the user specifies either of the `--build-only` or `--static-only` options for the installer. The default case of _not_ specifying any of those will not run the code inside this condition at all.

Reply:
for build or static, we run the code at lines 92-105
for native, we run the code at lines 1319-1332

I've tested the code and works as expected, in all cases.

Reply:
Have you tried running it on a system we do not provide native packages for _without_ specifying either `--build-only` or `--static-only`? Because unless I’m reading things wrong, with the code as currently written, neither this check, nor the one in the native package install code gets run in that case.